sway/commands.c

```diff
@@ -35,6 +35,8 @@
 #include "input.h"
 #include "border.h"
 
+#define ARRAY_SIZE(array) (sizeof(array) / sizeof(array[0]))
```

Comment:
Expand the macro everywhere. The size constant would have to be manually updated when we add new commands.